docs/redis.md

```diff
@@ -0,0 +1,34 @@
+# Usage of Redis for cozy-stack
+
+
+It will be important to figure out if we use a cluster of Redis or only one.
+Assuming a cluster allows us to scale better in the future but for now we will focus on a single redis target for the stack.
+
+## As a cache / store (KV)
+
+Already implemented in https://github.com/cozy/cozy-stack/blob/master/pkg/cache/cache.go
+
+To use with sessions / download manager
+
+
+## As a simple lock
+
+Usage ?
+
+ref
+https://redis.io/topics/distlock
+(algorithm for cluster)
+
+
+## As a FS lock (range+RW)
+
+Using a Range+RW lock is an optimization, it would be acceptable to use a simple lock instead to start.
```

Comment:
@nono for Range, the main gain I see is in extreme cases: I move my whole Picture folder and its 5k files to Trash. As an user, I expect the picture & trash folder to just give me a spinner for 5 minute. I do not expect I cant read my Documents folder. 

Anyway, I agree it will wait.